tests/integration/test_render.py

```diff
@@ -272,3 +266,32 @@ def test_zone_labels_shown(self):
         for zone in zones:
             zone_name = zone.find_element_by_css_selector('p.zone-name')
             self.assertNotIn('sr', zone_name.get_attribute('class'))
+
+
+@ddt
+class TestDragAndDropRenderZoneAlign(BaseIntegrationTest):
+    """
+    Verifying Drag and Drop XBlock rendering using zone alignment.
+    """
+    PAGE_TITLE = 'Drag and Drop v2'
+    PAGE_ID = 'drag_and_drop_v2'
+
+    def setUp(self):
+        super(TestDragAndDropRenderZoneAlign, self).setUp()
+        scenario_xml = self._get_custom_scenario_xml("data/test_zone_align.json")
+        self._add_scenario(self.PAGE_ID, self.PAGE_TITLE, scenario_xml)
+        self._page = self.go_to_page(self.PAGE_TITLE)
+
+    def test_zone_align(self):
+        self.assertEquals(self._get_style('#-zone-none .item-wrapper', 'textAlign'), 'start')
+        self.assertEquals(self._get_style('#-zone-none .item-wrapper', 'textAlign', True), 'start')
+        self.assertEquals(self._get_style('#-zone-invalid .item-wrapper', 'textAlign'), 'start')
+        self.assertEquals(self._get_style('#-zone-invalid .item-wrapper', 'textAlign', True), 'start')
+        self.assertEquals(self._get_style('#-zone-left .item-wrapper', 'textAlign'), 'left')
+        self.assertEquals(self._get_style('#-zone-left .item-wrapper', 'textAlign', True), 'left')
+        self.assertEquals(self._get_style('#-zone-right .item-wrapper', 'textAlign'), 'right')
+        self.assertEquals(self._get_style('#-zone-right .item-wrapper', 'textAlign', True), 'right')
+        self.assertEquals(self._get_style('#-zone-center .item-wrapper', 'textAlign'), 'center')
+        self.assertEquals(self._get_style('#-zone-center .item-wrapper', 'textAlign', True), 'center')
```

Comment:
@pomegranited I think it would be good to DRY this out a bit. How about:

```python
def test_zone_align(self):
    expected_alignments = {
        "#-zone-none": "start",
        "#-zone-invalid": "start",
        "#-zone-left": "left",
        "#-zone-right": "right",
        "#-zone-center": "center"
    }
    for zone_id, expected_alignment in expected_alignments.items():
        selector = "{zone_id} .item-wrapper".format(zone_id=zone_id)
        self.assertEquals(self._get_style(selector, "textAlign"), expected_alignment)
        self.assertEquals(self._get_style(selector, "textAlign", computed=True), expected_alignment)